Sources/ResilientDecoding/ErrorReporting.swift

```diff
@@ -234,12 +234,12 @@ public struct UnknownNovelValueError: Error {
   /**
    The raw value for which `init(rawValue:)` returned `nil`.
    */
-  public let novelValue: Any
+  public let novelValue: Sendable
```

Comment:
Worth noting that this is a breaking change, and we should update the sample dependency management code in the README to match.

Reply:
For sure, I see this being a minor version bump at a minimum.